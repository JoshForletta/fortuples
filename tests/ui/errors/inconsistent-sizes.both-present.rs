use fortuples::fortuples;

struct Container<T>(T);

fortuples! {
    #[tuples::min_size(2)]
    #[tuples::max_size(1)]
    impl Container<#Tuple> {}
}

fn main() {}
