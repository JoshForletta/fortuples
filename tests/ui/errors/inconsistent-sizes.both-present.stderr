error: `min_size` should be strictly less than `max_size`.
 --> tests/ui/errors/inconsistent-sizes.both-present.rs:6:15
  |
6 |     #[tuples::min_size(2)]
  |               ^^^^^^^^
